.github/workflows/func.yaml

```diff
@@ -171,11 +174,21 @@ jobs:
            else
              echo "Skip because missing openstack-auth-env secrets"
            fi
+      - name: pytest env
+        if: ${{ inputs.pytest-env }}
+        run: |
+           echo ${{ inputs.pytest-env }} | base64 -d | grep 'PYTEST_' >> "$GITHUB_ENV"
```

Comment:
I do not understand why this is needed. Why not simply run any test with `PYTEST_*=test tox -e func`? Can you provide example where this will be needed.

Reply:
https://github.com/canonical/charm-advanced-routing/blob/28162e0fb93b07f3894b00ad40d2093c12bbe234/src/tests/functional/conftest.py#L51

It's for some legacy testing. We need to provide the target cloud, region, and credential for it's functional testing.

Reply:
Those values could not be simply `None`?